src/analyses/static_analysis.cpp

```diff
@@ -65,14 +65,6 @@ void static_analysis_baset::operator()(
   fixedpoint(goto_functions);
 }
 
-void static_analysis_baset::operator()(
```

Comment:
We're using this one -- take an identifier rather than removing?

Reply:
ok